projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```diff
@@ -8,40 +8,32 @@
 @use '../../utils/mixins';
 @use '../../utils/variables/variables.global';
```

Comment:
We should be able to remove this `@use` now that all globals are replaced with local vars or tokens.

Reply:
Correct, but global `variables` file have map `clr-layers` that stores `z-index` level and istoo hard to use without it. It's used only here: 
`z-index: map.get(variables.$clr-layers, dropdown-menu);`

At later stage if there is CDS tokens about those levels we can use them.

Reply:
true, it remains used.